{
    "has_execution_error": true,
    "execution_output": "Fatal Python error: init_sys_streams: can't initialize sys standard streams\nPython runtime state: core initialized\nOSError: [Errno 9] Bad file descriptor\n\nCurrent thread 0x00000001dcac5c40 (most recent call first):\n  <no Python frame>\n",
    "pred": {
        "error": "malformed_output"
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "Laura",
                "start_time": "09:14",
                "end_time": "10:59"
            },
            {
                "action": "meet",
                "person": "Charles",
                "start_time": "11:30",
                "end_time": "12:15"
            },
            {
                "action": "meet",
                "person": "Margaret",
                "start_time": "14:15",
                "end_time": "16:15"
            },
            {
                "action": "meet",
                "person": "Rebecca",
                "start_time": "16:24",
                "end_time": "17:54"
            },
            {
                "action": "meet",
                "person": "Melissa",
                "start_time": "17:59",
                "end_time": "18:29"
            },
            {
                "action": "meet",
                "person": "Patricia",
                "start_time": "18:46",
                "end_time": "19:31"
            },
            {
                "action": "meet",
                "person": "Robert",
                "start_time": "19:46",
                "end_time": "20:16"
            },
            {
                "action": "meet",
                "person": "Karen",
                "start_time": "20:28",
                "end_time": "21:28"
            }
        ]
    },
    "status": "Execution error: malformed_output",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 1
}